cf/rds.yaml

```diff
@@ -37,6 +37,13 @@ Parameters:
     Type: String
     Default: aurora
 
+# --------------------------------------- Conditions ---------------------------------------
+
+Conditions: 
+  SandboxIdIsDefault: !Equals
+    - !Ref SandboxID
+    - default
```

Comment:
I haven't seen a CF function used this way before. I think you should use `!Equals [value_1, value_2]`

Reply:
Sure, just mentioning it's the same though, both are syntax for lists